Add to README about validating data (#9)
ShippingEasy/smart_enum: README.md

# => [["Toyota Camry", false], ["Freightliner Cascadia", true]]
```

### Validating data in SmartEnum

Currently there is no built in way to validate the data in SmartEnum.

> The pattern that we suggest is to add a small spec to your CI process to

Reviewer: ```suggestion
The pattern that we suggest is to add an automated test to
```
